src/main/java/objectsBasicsMandatoryPracticalTasks/Book.java

```diff
@@ -0,0 +1,17 @@
+package objectsBasicsMandatoryPracticalTasks;
+
+class Book {
+    private String title;
+    private double price;
+    Author author;
+
+    Book(String title, Author author, double price) {
+        this.title = title;
+        this.price = price;
+        author.printAuthor(author);
```

Comment:
No need to print author in constructor.
It should be initialized here and printed in `printBook()` method